lib/src/signed_video_h26x_common.c

```diff
@@ -844,6 +844,23 @@ update_gop_hash(void *crypto_handle, gop_info_t *gop_info)
   return status;
 }
 
+/* hash_the_hash_list()
+ * Takes all the hash nalus from |hash_list| and hash it.
+ */
+svrc_t
+hash_the_hash_list(signed_video_t *self)
```

Comment:
The explicit operation is "hashing the hash_list", but what you actually want to do is to compute the GOP hash. I suggest that you rename the function to `compute_gop_hash`